packages/app/src/cli/services/app-logs/write-app-logs.test.ts

```diff
@@ -42,6 +43,25 @@ describe('writeAppLogsToFile', () => {
     expect(writeLog).toHaveBeenCalledWith(expect.stringContaining(path), logData)
   })
 
+  test('calls writeLog with the right data with log_type !== undefined', async () => {
+    // Given
+    const appLog = {
+      ...APP_LOG,
+      log_type: 'function_run',
```

Comment:
Should this also set `event_type` to `undefined` to simulate the case where that's not returned anymore?

Reply:
Well technically the interpreter *should shortcut and not even check the `event_type` once it sees that `log_type` is defined & not null, so the value of `event_type` is not relevant for this test.